deepfence_frontend/apps/dashboard/src/queries/search.tsx

```diff
@@ -1272,6 +1278,21 @@ export const searchQueries = createQueryKeys('search', {
             'node_type'
           ] = serviceType;
         }
+        if (awsAccountId?.length) {
```

Comment:
What if the user selects a combination of account ids from different clouds?

Reply:
Rectified.